apps/vmq_server/src/vmq_subscriber.erl

```diff
@@ -173,13 +173,19 @@ get_node_subs(Node, Subs) ->
 -spec fold(function(), any(), subs() | changes()) -> any().
 fold(Fun, Acc, Subs) ->
     lists:foldl(fun({Node, _, NSubs}, AccAcc) ->
-                        lists:foldl(fun({Topic, QoS}, AccAccAcc) ->
-                                            Fun({Topic, QoS, Node}, AccAccAcc)
-                                    end, AccAcc, NSubs);
+                        lists:foldl(
+                          fun({Topic, QoS}, AccAccAcc) ->
+                                  Fun({Topic, QoS, Node}, AccAccAcc);
+                             ({Topic, QoS, Opts}, AccAccAcc) ->
```

Comment:
Rename QoS to SubInfo, and remove never matching case on linen 179

Reply:
Fixed